### ISSUE TITLE
[Bug] [1537] Reading position

### ISSUE THREAD

**Issue by rahmjan** [shosetsuorg/android-app#142]
Reading position is sometimes not set and the reading page after reopen is returned to the beginning of the page.

**Comment by c-h-u-c-k** [comment]
I agree, happens to me whenever the app is changed or the screen turns off.
